server/src/main/java/org/opensearch/index/merge/MergeStats.java

```diff
@@ -87,6 +91,11 @@ public MergeStats(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(Version.V_2_11_0)) {
             unreferencedFileCleanUpsPerformed = in.readOptionalVLong();
         }
+        if (in.getVersion().onOrAfter(Version.V_3_1_0)) {
```

Comment:
the version would the version where we intent to release this. 3.1 is already release. this should be ideally 3.3 now.